angular.json

```diff
@@ -125,5 +125,8 @@
         "@schematics/angular:directive": {
             "prefix": "app"
         }
+    },
+    "cli": {
+        "analytics": false
```

Comment:
I have to go through this every time I run npm commands and it has been a bit annoying.